lib/web/question_router.ts

```diff
@@ -984,9 +997,7 @@ export const questionRouter = router({
           title: `${submittedForecast.user.name || "Someone"} predicted on "${
             q.title
           }"`,
-          content: `${
-            submittedForecast.user.name || "Someone"
-          } predicted on ${getMarkdownLinkQuestionTitle(q)}`,
+          content: `${submittedForecast.user.name || "Someone"} predicted`,
```

Comment:
nit: what was the rationale behind this? I care more about predictions on some questions than others